intra_process_comms/CMakeLists.txt

```diff
@@ -11,6 +11,11 @@ find_package(rclcpp REQUIRED)
 find_package(rmw REQUIRED)
 find_package(rmw_implementation REQUIRED)
 find_package(std_msgs REQUIRED)
+find_package(sensor_msgs REQUIRED)
```

Comment:
k. I'll keep OpenCV in its own category instead of sorting it into alphabetical order, since it's not a ROS package (I assume that's our convention based on https://github.com/ros2/demos/blob/master/image_tools/CMakeLists.txt).